setup.py

```diff
@@ -172,9 +172,18 @@ def additional_commands():
         license='BSD',
         maintainer='ETS Developers',
         maintainer_email='enthought-dev@enthought.com',
-        package_data=dict(traitsui=['extras/images/*', 'image/library/*.zip',
-                                    'images/*', 'wx/images/*',
-                                    'qt4/images/*']),
+        package_data=dict(
+            traitsui=[
+                'examples/demo/*',
```

Comment:
do you think there's any advantage to explicitly listing the different sub dirs in `examples/demo/` which we are including or is that unnecessary?

Reply:
Seems unnecessary for now since we do want to include everything in there. If there is a reason to exclude some subdirectories and include some other directories, we can revisit this.